packages/cli/src/lib/platform/url.ts

```diff
@@ -0,0 +1,17 @@
+import {PlatformEnvironment, platformUrl} from './environment';
+
+export interface SnapshotUrlOptionsArgs<> {
```

Comment:
You don't need the `<>` I reckon?
```suggestion
export interface SnapshotUrlOptionsArgs {
```